packages/core/src/components/Dropdown/Dropdown.styles.js

```diff
@@ -358,15 +358,32 @@ const indicatorSeparator = () => () => ({
   display: "none"
 });
 
-const group = () => () => ({
-  paddingBottom: 0,
-  marginTop: "4px"
-});
+const group =
+  ({ withGroupDivider }) =>
+  () => ({
+    paddingBottom: 0,
+    marginTop: "4px",
+    ...(withGroupDivider && {
+      ":not(:last-child)": {
+        position: "relative",
+        paddingBottom: "8px",
+        "::after": {
+          content: '""',
+          position: "absolute",
+          bottom: "0",
+          left: "50%",
+          transform: "translateX(-50%)",
+          width: `calc(100% - (${getCSSVar("spacing-small")} * 2))`,
+          height: "1px",
+          backgroundColor: getCSSVar("layout-border-color")
```

Comment:
```suggestion
          backgroundColor: "var(layout-border-color)"
```
same as the above comments, shouldn't this work?